### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by upluke**
# Milestone: {Milestone 1, 3}
# Developer: Ke Lyu

### Pull Request Summary
Home page, about page and recipe pages are styled and completed. Please review and approve. thank you.
{Describe the purpose of your pull request}

### Pull Request Checklist
- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit message follows our guidelines
- [x] The milestone number is specified
- [x] The developer name is specified
- [x] The summary is completed

### Reviewer Checklist  - PULL REQUEST REVIEWER ONLY

IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized pull request reviewers. If you are the individual template contributor, do not fill out the rest of the fields or check the boxes.

NOTE: Milestones can only be completed when all boxes are checked.

- [x] The code is fully reviewed
- [x] Meaningful feedback is given
- [x] Comment that you have reviewed the code
- [x] This box is checked

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/styles.css">
```
Comment (vwinstea) [inline about.html]:
```suggestion
    <link rel="stylesheet" href="css/styles.css">
```

File: about.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>About</title>
+</head>
+<body>
+    <header> 
+        <nav class="nav ">
+            <div class="container">
+                <h1 class="logo"><a href="/index.html" class="current">MyKitchen</a> </h1>
+                <ul>
+                    <li><a href="/index.html" >Home</a></li>
```
Comment (vwinstea) [inline about.html]:
```suggestion
                    <li><a href="index.html" >Home</a></li>
```

**Review by vwinstea — CHANGES_REQUESTED** [review]
Unfortunately, it looks like you have run into a machine or browser specific issue on your site, so none of your links or css work on my computer. :( Luckily, it's a pretty easy fix, all you need to do is remove the leading`/` in all of your links. Please reach out on this PR or via slack if you have any questions.

Out of curiosity, what OS and browser are you using?

File: about.html (hunk 1 above)
Comment (upluke) [reply about.html]: Code updated! Thanks for the feedback. I use macOS Monterey and Google Chrome.

File: about.html (hunk 2 above)
Comment (upluke) [reply about.html]: Code updated!

**Comment by vwinstea** [comment]
Those fixes you made to the about page worked, but you'll have to fix the links on all of the other pages you made too since they all have the same issues

File: about.html (hunk 1 above)
Comment (upluke) [reply about.html]: > Done! Thanks for the feedback again.

**Review by vwinstea — COMMENTED** [review]
Much better, everything from your first milestone is working now! I love the styling you did on the recipe previews, they look great. Nice work!

**Comment by vwinstea** [comment]
Milestone 1 complete!

**Comment by vwinstea** [comment]
Could you @ me when you finish milestone 3, I just want to make sure this PR doesn't get buried in the other PRs that need to address their feedback still

**Comment by upluke** [comment]
milestone 1 finished

**Pull request closed without merging**